netbox/utilities/templatetags/helpers.py

```diff
@@ -100,7 +100,7 @@ def humanize_megabytes(mb):
         return f'{int(mb / 1048576)} TB'
     if not mb % 1024:
         return f'{int(mb / 1024)} GB'
-    return f'{mb} MB'
+    return f'{int(mb) / (1048576):.2f} MB'
```

Comment:
This logic needs to be rewritten as our intent is now to display the human-friendly value using the largest appropriate unit. Values of 1TB or larger should be displayed in fractional terabytes; values of 1GB or larger should be in gigabytes; and anything smaller should be in megabytes

Reply:
I rewrite the logic. I considered the MB to be coming in "Megabytes" because, in the VMs, the unit for memory is MB.